graph/src/data/subgraph/schema.rs

```diff
@@ -201,20 +202,53 @@ impl Display for SubgraphError {
     }
 }
 
+impl stable_hash_legacy::StableHash for SubgraphError {
+    fn stable_hash<H: stable_hash_legacy::StableHasher>(
+        &self,
+        mut sequence_number: H::Seq,
+        state: &mut H,
+    ) {
+        let SubgraphError {
+            subgraph_id,
+            message,
+            block_ptr,
+            handler,
+            deterministic,
+        } = self;
+        stable_hash_legacy::StableHash::stable_hash(
+            &subgraph_id,
```

Comment:
Nit: These `&` aren't be required because it's inferred by the destructuring from `&self`